feature-group-definitions/fonts.yml

```
@@ -1,4 +1,5 @@
 name: "@font-face"
+description: The `@font-face` CSS at-rule specifies a custom font with which to display text, which can be loaded from either a remote server or a locally-installed font on the device.
```

```suggestion
description: The `@font-face` CSS at-rule specifies a custom font with which to display text, which can be either loaded from a server or a font installed on the device.
```

---

```suggestion
description: The `@font-face` CSS at-rule specifies a custom font with which to display text, which can be either loaded from a server or installed on the device.
```

I think we can get rid of the repetition.

---

The problem I tried to avoid in my first suggestions is that it condenses down to "custom font can be loaded from a locally-installed font", and that "a remote server" and "locally-installed font" is an odd pair.

With the new wording it sounds like the rule has the effect of either loading a font or installing a font, which is also not right.

I'm not sure how to fix this. @ddbeck to the rescue?

---

Very good, let's have that one!